mbcrossval/crossvalidation.py

```diff
@@ -238,16 +233,19 @@ def initialization_selection():
     # use glacierwiede mu_star in order to finde t_star: it's faster!
     cfg.PARAMS['tstar_search_glacierwide'] = True
 
+    # set climate data
+    cfg.PARAMS['baseline_climate'] = mbcfg.PARAMS['baselineclimate'].upper()
+
     # Pre-download other files which will be needed later
-    _ = utils.get_cru_file(var='tmp')
-    _ = utils.get_cru_file(var='pre')
+    # TODO _ = utils.get_cru_file(var='tmp')
+    # TODO _ = utils.get_cru_file(var='pre')
```

Comment:
This can be safely removed I think

Reply:
I wasn't sure why this pre-download was necessary in the first place and wanted to think about it later. but yes pretty sure it can be removed.